packages/ui/src/services/utils.ts

```diff
@@ -1,6 +1,5 @@
 //  @ts-nocheck
 
-import { v4 as uuidv4 } from 'uuid'
```

Comment:
`uuid` module isn't specified in a package.json file
only in yarn.lock

Reply:
Yap. It is not going to final bundle. 👍